tests/conftest.py

```diff
@@ -198,6 +200,43 @@ def forecast_values():
     return forecast_values
 
 
+@pytest.fixture()
+def forecasts(db_session, sites):
+    """Make fake forecasts"""
+    init_timestamp = pd.Timestamp(dt.datetime.now(tz=None)).floor(dt.timedelta(minutes=15))
+
+    n = 24 * 4  # 24 hours of readings of 15
+    start_times = [init_timestamp - dt.timedelta(minutes=x * 15) for x in range(n)]
+    start_times = start_times[::-1]
+
+    for site in sites:
+        forecast_uuid = uuid4()
+        model = get_or_create_model(db_session, "test", "0.0.0")
+        forecast = ForecastSQL(
+            site_uuid=site.site_uuid,
+            timestamp_utc=start_times[-1],
+            forecast_version="0.0.0",
+            created_utc=start_times[-1],
+            forecast_uuid=forecast_uuid,
+        )
+        db_session.add(forecast)
+
+        forecast_values = []
+        for i in range(0, len(start_times)):
+            forecast_value = ForecastValueSQL(
+                horizon_minutes=i * 15,
+                forecast_power_kw=random.random() * 10000,
```

Comment:
Would it be easier to test/ have stronger tests if these weren't random values? Thinking then some of the != asserts in tests below could be stronger = asserts?

Reply:
your right, they would be better. Trying to work out if to do it in this PR, or put it as a good first issue. The generation values are currently random too, so would need to change them too

Reply:
https://github.com/openclimatefix/india-forecast-app/issues/119